.github/workflows/release.yml

```diff
@@ -9,77 +9,93 @@ jobs:
     runs-on: ${{ matrix.operating-system }}
     strategy:
       matrix:
-        operating-system: [ubuntu-20.04, windows-latest, macos-11]
+        include:
+          - operating-system: ubuntu-latest
```

Comment:
We shouldn't run latest (apart from Windows) as GitHub could update those and that can cause issues.

Reply:
Why not for Windows? `windows-latest` currently points to `windows-2022` and the other version available is `windows-2019`.

Reply:
Windows Win32API is very stable. It doesn't seem to matter what version you build on, it will run on older versions. But if you want to be consistent ...